task_sdk/src/airflow/sdk/definitions/baseoperator.py

```diff
@@ -869,6 +874,11 @@ def __init__(
 
         validate_instance_args(self, BASEOPERATOR_ARGS_EXPECTED_TYPES)
 
+        # Ensure priority_weight is within the valid range
+        self.priority_weight = max(
+            MINIMUM_PRIORITY_WEIGHT, min(MAXIMUM_PRIORITY_WEIGHT, self.priority_weight)
+        )
```

Comment:
Shoud we use `db_safe_priority` here instead of re-impl?

Reply:
Would like to but the implementaiton currently is in `airflow-utils` - I asusme we want to have task SDK split apart from core?

Still what I can do is implement the same in task-SDK as utility... the old abstract/base operator in core will go away once AIP-72 is complete, correct?

Reply:
There's still loads of cross imports to fix up, so for now I'd say have one version

Reply:
Okay, if it is okay to keep it coupled for the moment... consolidated the calls.